src/main/java/io/anserini/ann/EvalInvertedDenseVectors.java

```diff
@@ -53,7 +55,7 @@
 
 import static org.apache.lucene.search.BooleanClause.Occur.SHOULD;
 
-public class ApproximateNearestNeighborEval {
+public class EvalInvertedDenseVectors {
```

Comment:
I think the point was to get rid of the `io.anserini.ann` package... so maybe move this to `io.anserini.search` - and rename to `Evaluate`? Typically, we'd write eval util classes in Python...

Reply:
ok, if we're to remove the `ann` package we also need to move `FeatureVectorTokenizer`,  and both`fw` and `lexlsh' packages, e.g., into `io.anserini.analysis` (as they are tokenizers+filters in the end).

Reply:
Agreed.